src/GitVersion.BuildAgents/Agents/BitBucketPipelines.cs

```diff
@@ -10,6 +10,7 @@ internal class BitBucketPipelines : BuildAgentBase
     public const string TagEnvironmentVariableName = "BITBUCKET_TAG";
     public const string PullRequestEnvironmentVariableName = "BITBUCKET_PR_ID";
     private string? file;
+    private const string Ps1File = "gitversion.ps1";
```

Comment:
better to have a similar approach as with property file
Set the Ps1File with a `WithPowershellFile` method, and call it in the constructor

Reply:
Fixed.